client/src/containers/App/App.js

```diff
@@ -20,6 +22,8 @@ class App extends Component<Props> {
         <Route path="/login" component={Login} />
         <Route path="/signup" component={Signup} />
         <Route path="/forgot-password" component={ForgotPassword} />
+        <Route path="/events" component={Events} />
+        <Route path="/details/:value" component={EventDetails} />
```

Comment:
for event details we will have events/:event_id

Reply:
was changed it but those changes are in another PR.